Test branch

rakhmatillaa opened the pull request.

rakhmatillaa committed:
added first exercise

rakhmatillaa committed:
added second exercise

rakhmatillaa committed:
added third exercise

rakhmatillaa committed:
added fourth exercise

this exercise is not completely correct

JmlKhan commented on exercise2.cs:
```diff
@@ -0,0 +1,46 @@
+
+    static void Main(string[] args)
+    {
+        List<string> wordList = new List<string>();
```
try to add list items at once

JmlKhan commented on exercise4.cs:
```diff
@@ -0,0 +1,22 @@
+    static void Main(string[] args)
+    {
+        List<string> list = new List<string>();
+        list.Add("abc");
```
same here as well

JmlKhan commented on exercise1.cs:
```diff
@@ -0,0 +1,28 @@
+            Console.WriteLine(u);
+        }
+        Console.ReadLine();
+    }
```
add space before function declaration

JmlKhan commented on exercise4.cs:
```diff
@@ -0,0 +1,22 @@
+        {
+            Console.WriteLine(item);
+        }
+
```
remove space here

rakhmatillaa committed:
added spaces

rakhmatillaa committed:
added spaces and shortened the code

rakhmatillaa committed:
Update exercise3.cs

rakhmatillaa committed:
corrected some mistakes

rakhmatillaa committed:
Update exercise2.cs

rakhmatillaa committed:
made the code shorter,but still has a problem

couldn't find out how to solve a problem with printing only unique(non-duplicate) strings,need some help
